spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #show' do
+    it 'returns a successful response' do
+      user = User.create(name: 'paul', photo: 'https://pbs.twimg.com/media/FfhkDtjWQAMYNrc.jpg:large', bio: 'Teacher from Burundi.',
+                         posts_counter: 0)
+      get user_path(user.id)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      user = User.create(name: 'paul', photo: 'https://pbs.twimg.com/media/FfhkDtjWQAMYNrc.jpg:large', bio: 'Teacher from Burundi.',
+                         posts_counter: 0)
+      get user_path(user.id)
+      expect(response).to render_template(:show)
+    end
```

Comment:
- [ ] The same happens here, the `response.body` test is not present. Please make sure to include it! 🤓 🔧